torch/csrc/distributed/c10d/ProcessGroup.hpp

```diff
@@ -505,6 +510,7 @@ class TORCH_API ProcessGroup : public torch::CustomClassHolder {
     // TODO: HACK for backend name to get sequence number for that backend.
     if (backendType == ProcessGroup::BackendType::GLOO ||
         backendType == ProcessGroup::BackendType::NCCL ||
+        backendType == ProcessGroup::BackendType::XCCL ||
         backendType == ProcessGroup::BackendType::UCC) {
```

Comment:
Should we invert the conditional? Or perhaps add `inline bool backendSupportsSequenceNumbers()` and call it here and on line 533 as well?

Reply:
Yes, make sense to put the condition check into a function instead of a long if ...else.